WalletWasabi.Fluent/Helpers/TextHelpers.cs

```diff
@@ -9,7 +9,7 @@ namespace WalletWasabi.Fluent.Helpers;
 
 public static partial class TextHelpers
```

Comment:
This text helper is not very good

Reply:
I would prefer not to change anything functional in this PR if possible